Makefile.js

```diff
@@ -978,7 +978,7 @@ target.checkGitCommit = function() {
     }
 
     // Only check non-release messages
-    if (!semver.valid(commitMsgs[0]) && !/^Revert /.test(commitMsgs[0])) {
+    if (!semver.valid(commitMsgs[0]) && !commitMsgs[0].startsWith("Revert")) {
```

Comment:
This may or may not matter, but this actually changed the semantics of the code. Previously it was looking for strings starting with `Revert ` (with a space), now it looks for just `Revert`. This will catch things that it wouldn't have before, such as `Reverted` or `Reverting`.

Reply:
Probably not good! Mind filing a PR to correct? Thanks!